registry/openjdk/container.yaml

```diff
@@ -4,13 +4,18 @@ maintainer: '@vsoch'
 description: OpenJDK is an open-source implementation of the Java Platform, Standard
   Edition.
 latest:
-  17.0.2: sha256:2c3ffd17a95947a187cbf00b454656566063cc53f21ea419929c0d23f3347572
+  19-ea-15: sha256:7e593cb75001103d515e7f3a1fc81598c5d7753339368399b7b5226816036c7f
 tags:
   16.0.1-buster: sha256:61f3786a28ed911028f8e7e3b65a57a8a9ed04067d137317d369c9b3bc11b289
   16.0.2: sha256:bb68f084c2000c8532b1675ca7034f3922f4aa10e9c7126d29551c0ffd6dee8f
   17.0.1: sha256:0da39ed69dec14f9603e2b916592691cc39341510abdf4255abb1c90b00eb3f4
-  17.0.2: sha256:2c3ffd17a95947a187cbf00b454656566063cc53f21ea419929c0d23f3347572
-  latest: sha256:2c3ffd17a95947a187cbf00b454656566063cc53f21ea419929c0d23f3347572
+  17.0.2: sha256:3d88de143bdd54d3e873302355564818e0ac0d17dbacc9869090197b7dd8a8f3
+  latest: sha256:3d88de143bdd54d3e873302355564818e0ac0d17dbacc9869090197b7dd8a8f3
+  '19': sha256:7e593cb75001103d515e7f3a1fc81598c5d7753339368399b7b5226816036c7f
+  19-ea-10: sha256:1dceb32239eb8a2558be7bbc6fcf53b9759799b99a75f1f3a73f483cc79ac028
+  19-alpine3.15: sha256:fd8c856fb7e3fc2d93815c4b9fa1d112c8d9f46d01f64c6ea858d33518555e5f
+  19-ea-15: sha256:7e593cb75001103d515e7f3a1fc81598c5d7753339368399b7b5226816036c7f
+  19-ea-14: sha256:ad5eab9ef6f4e4d0dd971ea0e94eaf992f4d8fb1853ee5d8dfac7e273f4dfed3
```

Comment:
I'm not actively developing with Java, but as far as I can see, `-ea` means "early access". Would it make sense to completely filter them out ? Otherwise, I think there should ideally only be the last one, i.e. `19-ea-15`, but it's probably not worth modifying the shpc code for that

Reply:
Ah we can add a custom filter here to avoid tags with ea!